src/Std/Classes/Ord.lean

```diff
@@ -435,4 +457,248 @@ theorem lawfulBEq_of_lawfulEqOrd [Ord α] [LawfulEqOrd α] : LawfulBEq α where
 
 end Internal
 
+section Instances
+
+theorem TransOrd.of_lt_trans_of_lt_iff {α : Type u} [LT α] [DecidableLT α] [DecidableEq α]
```

Comment:
The names seems to be quite off. More like `transCmp_compareOfLessAndEq_of_...` or `TransCmp.compareOfLessAndEq_...`?